### DISCUSSION TITLE
Increase currency field (uitype: 71) length?

### DISCUSSION THREAD

**Discussion by rochmirabeau** [YetiForceCompany/YetiForceCRM#15341] [How to] [answered]
I've created a number of currency fields that I need to make longer (they are currently 99999).

I've tried editing the decimal (length/values) in the structure of the u_yf_*MODULE* table ( to 25,8), but that doesn't seem to have an effect.

**Comment by rskrzypczak** [comment, marked as answer]
> I've tried editing the decimal (length/values) in the structure of the u_yf_MODULE table.. 

ok this is the first step.
Now change the value in the maximumlength column of the vtiger_field table.

**Reply by rochmirabeau** [reply]
Done and done. Many thanks!
